src/test/java/su/litvak/chromecast/api/v2/MediaStatusTest.java

```diff
@@ -74,7 +74,7 @@ public void testDeserializationWithChromeCastAudioFixture () throws Exception {
         assertEquals(PlayerState.BUFFERING, mediaStatus.playerState);
         assertEquals(RepeatMode.REPEAT_OFF, mediaStatus.repeatMode);
         assertEquals(15, mediaStatus.supportedMediaCommands);
-        assertEquals(new Volume(1f, false), mediaStatus.volume);
+        assertEquals(new Volume(1f, false, Volume.default_increment), mediaStatus.volume);
```

Comment:
I guess you don't check the `increment` value here because in JSON it is `0.04` while the `default_increment` is `0.05`. I believe you should update `equals()` and `hashCode()` methods in `Volume` class so they will include `increment` property.